bin/nav

```diff
@@ -219,6 +251,27 @@ def c_config(_):
                  "{}".format('\n'.join(CONFIG_LOCATIONS)))
 
 
+def command_config_path(_):
+    """Prints the list of file system locations NAV will search for config"""
+    from nav.config import CONFIG_LOCATIONS
+    for path in CONFIG_LOCATIONS:
+        print(path)
+
+
+def command_config_install(parser):
```

Comment:
Does this need the same type of arg as `c_version(_):`? Github won't allow me to add a comment where the **c_**-block starts.

If they all take the same type of argument maybe the block should be marked out with a comment.

Maybe `_` as standard dummy argname should be documented in a style guide too.

Reply:
Because that code part is outdated from my latest push, I guess. Our style-guide consists of a finger that points to PEP-8 as of today, but you're welcome to draft a new version :)